core/src/inputs/mod.rs

```diff
@@ -29,7 +32,6 @@ pub use advice::AdviceSet;
 /// TODO: add more detailed explanation.
 #[derive(Clone, Debug)]
 pub struct ProgramInputs {
-    stack_init: Vec<Felt>,
```

Comment:
In the next PR we should rename this to `AdviceInputs` and move it to the `processor` crate.